apple_pay_card.go

```diff
@@ -12,6 +12,7 @@ type ApplePayCard struct {
 	CardType              string         `xml:"card-type"`
 	PaymentInstrumentName string         `xml:"payment-instrument-name"`
 	SourceDescription     string         `xml:"source-description"`
+	Bin                   string         `xml:"bin,omitempty"`
```

Comment:
Would that imply to also change it in `CreditCard`?, and at the same time you probably want to change also consider `CustomerId` in other structs since it is being shown as a warning by my linter already.

Reply:
I don't think we should update those things right now. New fields we've been making them correctly and old fields we need a way of updating then and keeping backwards compatibly, but that should happen in a different PR.